InnerEye/ML/configs/histo_configs/classification/BaseMIL.py

```diff
@@ -47,6 +49,10 @@ class BaseMIL(LightningContainer):
                                                   "dataset upfront and save it to disk.")
     # local_dataset (used as data module root_path) is declared in DatasetParams superclass
 
+    # slide dataset parameters:
+    slide_datatype: Optional[str] = param.String(default=None, allow_None=True, doc="Name of the slide dataset class if available.")
```

Comment:
not sure I understand this parameter - maybe rename to slide_dataclass and give an example in the doc